clarity_ext/extensions.py

```diff
@@ -226,7 +226,7 @@ def parse_run_argument(in_argument):
                 raise ValueError("Unexpected run argument type")
             return test
 
-        instance = self._get_extension(module)(None)
+        instance = self._get_extension(module)(None, None, None)
```

Comment:
This extra call to the extension has always confused me. It also makes it awkward to use the init() methods in extensions, if one should want to. Anyway, this is perhaps out of scope.